src/Features/CSharp/Portable/IntroduceUsingStatement/CSharpIntroduceUsingStatementCodeRefactoringProvider.cs

```diff
@@ -46,16 +46,17 @@ protected override SyntaxNode WithStatements(SyntaxNode parentOfStatementsToSurr
                 throw ExceptionUtilities.UnexpectedValue(parentOfStatementsToSurround);
         }
 
-        protected override StatementSyntax CreateUsingStatement(LocalDeclarationStatementSyntax declarationStatement, SyntaxTriviaList sameLineTrivia, SyntaxList<StatementSyntax> statementsToSurround)
+        protected override StatementSyntax CreateUsingStatement(LocalDeclarationStatementSyntax declarationStatement, SyntaxList<StatementSyntax> statementsToSurround)
         {
             var usingStatement = SyntaxFactory.UsingStatement(
+                SyntaxFactory.Token(SyntaxKind.UsingKeyword).WithLeadingTrivia(declarationStatement.GetLeadingTrivia()),
+                SyntaxFactory.Token(SyntaxKind.OpenParenToken),
                 declaration: declarationStatement.Declaration.WithoutTrivia(),
                 expression: null, // Declaration already has equals token and expression
+                SyntaxFactory.Token(SyntaxKind.CloseParenToken).WithTrailingTrivia(declarationStatement.GetTrailingTrivia()),
                 statement: SyntaxFactory.Block(statementsToSurround));
```

Comment:
logic now is much more sensible.  The trivia around the initial declstatement get moved to the `using (...)` portion of the using.